docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,505 @@
+.. _schema_registry_tutorial:
+
+Confluent Schema Registry Tutorial
+==================================
+
+This tutorial provides an example on how to use |sr-long|.
+
+**Contents**
+
+.. contents::
+  :local:
+  :depth: 2
+
+
+Overview
+~~~~~~~~
+
+This tutorial provides a step-by-step example to use |sr-long|.
+It walks through the steps to enable client applications to read and write Avro data with compatibility checks as schemas evolve.
+
+Benefits
+^^^^^^^^
+
+Kafka producers write data to Kafka topics and Kafka consumers read data from Kafka topics.
+There is an implicit "contract" that producers write data with a schema that can be read by consumers, even as producers and consumers evolve their schemas.
+
+It is useful to think about schemas as APIs.
+It is critical that applications built on top of those APIs continue to work even if the APIs have changed.
+Similarly, schema evolution requires compatibility checks to ensure that the producer-consumer contract is not broken. 
+This is where |sr-long| helps: it provides centralized schema management and compatibility checks as schemas evolve.
+
+For a more in-depth understanding of the benefits of Avro and |sr|, please refer to the following resources:
+
+* `Why Avro For Kafka Data <https://www.confluent.io/blog/avro-kafka-data/>`_
+* `Yes, Virginia, You Really Do Need a Schema Registry <https://www.confluent.io/blog/schema-registry-kafka-stream-processing-yes-virginia-you-really-need-one/>`_
+
+Target Audience
+^^^^^^^^^^^^^^^
+
+The target audience is a developer writing Kafka streaming applications, who wants to build a robust application leveraging Avro data and |sr-long|.
+
+This tutorial is not meant to cover the operational aspects of running the |sr| service. For production deployments of |sr-long|, please refer to :ref:`Schema Registry Operations<schemaregistry_operations>`.
+
+Before You Begin
+~~~~~~~~~~~~~~~~
+
+Prerequisites
+^^^^^^^^^^^^^
+
+Before proceeding with this tutorial
+
+1. Verify have installed on your local machine:
+
+* Java 1.8 to run |cp|
+* Maven to compile the client Java code
+* `jq` tool to make output from SR REST endpoint pretty
+
+2. Use the :ref:`quickstart` to bring up |cp|. With a single-line command, you can have running on your local machine a basic Kafka cluster with |sr-long| and other services.
+
+.. sourcecode:: bash
+
+   $ confluent start
```

Comment:
please remove `$` from the commands.

Reply:
@joel-hamill : if I remove `$` from commands, how do we distinguish command from the output of issuing the command?